docs/examples/pipelinerun_image.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: steward.sap.com/v1alpha1
+kind: PipelineRun
+metadata:
+  generateName: image-
+spec:
+  jenkinsfileRunner:
+    image: stewardci/stewardci-jenkinsfile-runner:TEST_201006_b4e6c67
```

Comment:
Test images are cleaned from time to time. Use a released one.

Reply:
Changed